spec/models/articles_db_view_spec.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ArticlesDbView, type: :model do
+  
+  tag_set = %w(work робота Work Робота)
```

Comment:
what is this?

Reply:
This is an array of tags. Here it is used to check all occurrences of mentioned tags. Used to automate testing proces

Reply:
all tags in the database are written in lowercase

[https://github.com/mmrshk/innovation_forpost/blob/fa8e7ebda1cbabc0b2abddc19e6e721f9b4918de/app/forms/articles/create_update_form.rb#L38-L41](https://github.com/mmrshk/innovation_forpost/blob/fa8e7ebda1cbabc0b2abddc19e6e721f9b4918de/app/forms/articles/create_update_form.rb#L38-L41)